pkg/virt-handler/node-labeller/cpu_plugin.go

```diff
@@ -189,6 +189,15 @@ func (n *NodeLabeller) getDomCapabilities() (HostDomCapabilities, error) {
 	domCapabilitiesFile := filepath.Join(n.volumePath, n.domCapabilitiesFileName)
 	hostDomCapabilities := HostDomCapabilities{}
 	err := n.getStructureFromXMLFile(domCapabilitiesFile, &hostDomCapabilities)
+	if err != nil {
+		return hostDomCapabilities, err
+	}
+
+	if hostDomCapabilities.SEV.Supported == "yes" && hostDomCapabilities.SEV.MaxESGuests > 0 {
```

Comment:
Is SEV struct always set or can it be nil?

Reply:
I tested this by removing the entire sev node from `pkg/virt-handler/node-labeller/testdata/domcapabilities_sev.xml` and the SEV struct is still not nil.